cypress/integration/my_patches.ts

```diff
@@ -12,10 +17,10 @@ describe("My Patches Page", () => {
   beforeEach(() => {
     cy.preserveCookies();
     cy.listenGQL();
-    cy.visit(MY_PATCHES_ROUTE);
   });
 
   it("Typing in patch description input updates the url, requests patches and renders patches", () => {
+    cy.visit(MY_PATCHES_ROUTE);
```

Comment:
why do cy.visit in beginning of each test instead of doing it in the `beforeEach` ?

Reply:
cy.visit was being called before each test in nested describe which didn't work well with the page changing tests